packages/patternfly-4/react-integration/cypress/integration/chipgroupdefaultisopen.spec.ts

```diff
@@ -0,0 +1,36 @@
+describe('Chip Group Demo Test', () => {
+    it('Navigate to demo section', () => {
+      cy.visit('http://localhost:3000/');
+      cy.get('#chipgroup-default-is-open-demo-nav-item-link').click(); 
+      cy.url().should('eq', 'http://localhost:3000/chipgroup-default-is-open-demo-nav-link');
+    });
+  
+    it('Verify chip default text', () => {
```

Comment:
shouldn't we be verifying the chip group is open?

Reply:
Yes, nice catch! fixed it 👍